### ISSUE TITLE
The d.ts declaration file can be accessed in the ts file, Cannot find in the vue file

### ISSUE THREAD

**Issue by stempest0925** [vuejs/vue-loader#1752]
### Version
15.9.3

### Reproduction link
[https://github.com/HuijiFE/void-ui/tree/0.1](https://github.com/HuijiFE/void-ui/tree/0.1)

### Steps to reproduce
Modifying the tsconfig.json file multiple times to no avail

### What is expected?
Able to access the types defined globally in d.ts in the vue file

### What is actually happening?
Cannot find in the vue file

---
none

**Comment by sodatea** [comment]
The reproduction link returns a 404 page.

**Comment by stempest0925** [comment]
I installed ts loader and configured some things. After restarting the editor, I found that it was OK, but when I was looking for accurate configuration of those things, I rolled back, deleted dependencies, restarted the editor, or even created a new project, and found that the problem would no longer occur. . . in the before, It has always been a module that cannot access the ts declaration file under the vue file. . . .
